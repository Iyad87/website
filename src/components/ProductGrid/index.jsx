import React from 'react';

const ProductGrid = () => <div />;

export default ProductGrid;
