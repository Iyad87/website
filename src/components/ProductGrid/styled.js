import styled from 'styled-components';

export const Wrapper = styled.div`
  background: #ffffff;
  border-radius: 4px;
  box-shadow: 0 0 10px #a1a0c5;
`;
