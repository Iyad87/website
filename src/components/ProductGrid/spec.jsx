import React from 'react';
import { shallow } from 'enzyme';
import ProductGrid from './index';

test('components/ProductGrid snapshot', () => {
  const wrapper = shallow(<ProductGrid />);

  expect(wrapper).toMatchSnapshot();
});
