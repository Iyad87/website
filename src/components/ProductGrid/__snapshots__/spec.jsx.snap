// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/ProductGrid snapshot 1`] = `<div />`;
